app/routes/task_routes.py

```diff
@@ -1 +1,109 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint,request,abort,make_response,Response
```

Comment:
It takes more time to read a line when we have to break up the text a bit ourselves, adding spaces between the imports makes a difference when reading through a lot of code.
```suggestion
from flask import Blueprint, request, abort, make_response, Response
```